specification/ai/Face/models.session.tsp

```diff
@@ -8,12 +8,15 @@ using TypeSpec.Rest;
 
 namespace Face;
 
-@doc("The operation mode for the liveness modal.")
+@doc("The liveness operation mode to drive the client’s end-user experience.")
 union LivenessOperationMode {
   string,
 
-  @doc("The operation mode for the liveness modal.")
+  @doc("Utilizes a passive liveness technique that requires no additional actions from the user. Requires normal indoor lighting and high screen brightness for optimal performance. And thus, this mode has a narrow operational envelope and will not be suitable for scenarios that requires the end-user’s to be in bright lighting conditions. Note: this is the only supported mode for the Mobile (iOS and Android) solution.")
   "Passive",
+
+  @doc("This mode utilizes a hybrid passive or active liveness technique that necessitates user cooperation. It is optimized to require active motion only under suboptimal lighting conditions. Unlike the passive mode, this mode has no lighting restrictions, and thus offering a broader operational envelope. This mode is preferable on Web based solutions due to the lack of automatic screen brightness control available on browsers which hinders the Passive mode’s operational envelope on Web based solutions.")
+  "PassiveActive",
```

Comment:
`PassiveActive` is already in our service and has been there from the beginning. We wanted to hide though since the other component was not ready yet. It should be published for now.